src/People.tsx

```diff
@@ -0,0 +1,164 @@
+import { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { Loader } from './components/Loader';
+import { getPeople } from './api';
+
+import './App.scss';
+import { Person } from './types';
+import { PersonLink } from './PersonLink';
+
+export const People = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [loading, setloading] = useState(false);
+  const [err, setErr] = useState(false);
+  const [dataComes, setDataComes] = useState('');
+
+  const { personName } = useParams();
+  const selectedTab = personName;
```

Comment:
As i said before - it is better to use `slug` as variable name, and you don't need a `selectedTab` variable.